addons/website/views/snippets/snippets.xml

```diff
@@ -77,7 +77,10 @@
                 <t t-snippet="website.s_embed_code" string="Embed Code" t-forbid-sanitize="true"
                     t-image-preview="/website/static/src/img/snippets_previews/s_embed_code_preview.png" t-thumbnail="/website/static/src/img/snippets_thumbs/s_embed_code.svg" group="content"/>
                 <t t-snippet="website.s_numbers" string="Numbers" group="content">
-                    <keywords>statistics, stats, KPI</keywords>
+                    <keywords>statistics, stats, KPI, metrics, dashboard, analytics, highlights, figures, performance, achievements, benchmarks, milestones, indicators, data, measurements, reports, trends, results</keywords>
+                </t>
```

Comment:
Out of scope for this PR. There is already `task-4063510` handling this ;)

Reply:
I dropped this change. Tell me if i can move to TR :)